airbyte-integrations/connectors/destination-shelby/src/main/kotlin/http/job/JobStatus.kt

```diff
@@ -0,0 +1,18 @@
+package io.airbyte.integrations.destination.shelby.http.job
+
+enum class JobStatus {
```

Comment:
This feels somewhat redundant with the other set of states from the JobRepository. Maybe we could merge them?